AirCasting/APICommunicator/DormantStreamAlertAPI.swift

```diff
@@ -0,0 +1,70 @@
+// Created by Lunar on 28/07/2022.
+//
+
+import Foundation
+import Resolver
+
+enum DormantStreamAlertAPIError: Error {
+    case wrongRequest
+    case failedRequest(Error)
+}
+
+struct DormantStreamAlertAPI {
+    @Injected private var urlProvider: URLProvider
+    @Injected private var apiClient: APIClient
+    @Injected private var validator: HTTPResponseValidator
+    @Injected private var authorization: RequestAuthorisationService
+    
+    private let encoder = JSONEncoder()
+    private let decoder = JSONDecoder()
+    
+    struct Params: Encodable {
+        private let data: Nested
+        
+        struct Nested: Encodable {
+            let sessionStoppedAlert: String
+        }
+        
+        init(value: Bool) {
+            self.data = Nested(sessionStoppedAlert: String(value))
+        }
+    }
+    
+    struct Output: Decodable {
+        let action: String
+    }
+    
+    func sendNewSetting(value: Bool, completion: @escaping (Result<Void, DormantStreamAlertAPIError>) -> Void) {
+        let url = urlProvider.baseAppURL.appendingPathComponent("api/user/settings")
+        
+        let params = Params(value: value)
+        var request = URLRequest(url: url)
+        request.httpMethod = "POST"
+        request.addValue("application/json", forHTTPHeaderField: "Content-Type")
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+        
+        do {
+            request.httpBody = try encoder.encode(params)
+            _ = try authorization.authorise(request: &request)
```

Comment:
I get a warning if I don't add this. Do you know if there is a way around this?

Reply:
I said it to you, becouse I was not getting any warning when deleting `_ =`. Please skip then, I think it would be a waste of time to think about this